README.rst

```diff
@@ -51,6 +51,14 @@ System requirements:
 - gobject-introspection type libraries for GTK+, Pango
 - GTK+ 3.10 or newer (3.14 or newer for best results)
 
+On Debian/Ubuntu, the following packages may need installation:
+- python-docutils
+- gettext
+- python-pip
```

Comment:
Is pip required?  I'm surprised.

If you install python-gobject and python-cairo, you should be able to run gtimelog from a source checkout without pip installing it.